content/spin/writing-apps.md

```diff
@@ -263,7 +263,7 @@ files = [ "images/**/*.jpg", { source = "styles/dist", destination = "/styles" }
 
 The `files` field is an array listing the files, patterns and directories you want to include. Each element of the array can be:
 
-* A glob pattern, such as `images/**/*.jpg`.  In this case, the files that match the pattern are available to the Wasm code, at the same paths as they are in your file system. For example, if the glob pattern matches `images/photos/lake.jpg`, the Wasm module can access it using the path `/images/photos/lake.jpg`.  Glob patterns are relative to the directory containing `spin.toml`, and must be within that directory.
+* A glob pattern, such as `images/**/*.jpg`.  In this case, the files that match the pattern are available to the Wasm code, at the same paths as they are in your file system. For example, if the glob pattern matches `images/photos/lake.jpg`, the Wasm module can access it using the path `images/photos/lake.jpg`.  Glob patterns are relative to the directory containing `spin.toml`, and must be within that directory.
```

Comment:
In the guest, globbed files retain their position relative to the app directory, but transplanted to be relative to `/`... which is also the guest working directory.  So the original text is right... and this text is also right (provided the guest doesn't change current directory, which I don't know if that is even a thing for Wasm).

Pointing people at the relative path is probably better because the whole point of globs was that you would get the same behaviour in native (e.g. unit tests) as in Wasm.  And the path in the original text would _not_ work in native.

Pardon my ramble.  Just trying to think this through but I think I agree!

Reply:
This makes sense! Thanks Ivan!